docs/source/examples.ipynb

```diff
@@ -0,0 +1,50 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Examples\n",
+    "\n",
+    "Here is a list of examples using Mitiq for research.\n",
+    "\n",
+    "`[Example here](../../examples/research_example.ipynb)`"
```

Comment:
Also no examples/ dir.

Reply:
Thanks. Somehow had to fiddle with it for git to see it, probably as I stashed related commits earlier. I added ipynb checkpoints to the gitignore file in relation to keeping this clean. Also a readme file in the directory.